### PR TITLE
Inference using engine

### PR DESCRIPTION
**Pull request by MateoLostanlen**
The goal of this PR is to update the api inference function by using the pyronear engine to run the prediction

This PR is not ready yet, we are juste sharing progress with @fe51 do not review it :)

### CODE REVIEW

File: server/docker-compose.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ services:
     build:
       context: ..
       dockerfile: ./server/src/Dockerfile
-    command: uvicorn server.src.app.main:app --reload --workers 1 --host 0.0.0.0 --port 8000
+    command: uvicorn server.src.app.main:app --reload --workers 1 --host 0.0.0.0 --port 80
```
Comment (fe51) [inline server/docker-compose.yml:8]: Just for reference, why 80 instead of 8000 ?

File: pizero/runner.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import io
+import time
+import picamera
+from PIL import Image
+import requests
+
+
+url = "http://192.168.1.62:8000/file" # api url
```
Comment (fe51) [inline pizero/runner.py:8]: Considering, the change in docker compose, should not it be 80 ?

Comment (fe51) [reply pizero/runner.py:8]: So far, okay for the url as it is, but will be in the end in a config file or var env ! (I might came with a suggestion soon, a "fleet-config-file" to refer to (not versionned in the repo), and the url would based on those info)

Comment (fe51) [reply pizero/runner.py:8]: Also, does it work as it is ? 

as long as in `server/src/app/main.py` , l19, the inference route is prefixed as inference ` prefix="/inference`, should not it be
url = "http://192.168.1.62:8000/inference/file" ?

**Review by fe51 — COMMENTED** [review]
Hey ! Thanks for the PR and pleased to see expected features coming up !

I have left some comments and questions !

File: server/docker-compose.yml (hunk 1 above)
Comment (MateoLostanlen) [reply server/docker-compose.yml:8]: no specific reason, it wasn't on purpose. juste I didn't put back the right port after testing stuff. I will move everything back to 8000

File: pizero/runner.py (hunk 2 above)
Comment (MateoLostanlen) [reply pizero/runner.py]: Yes I need to a something to proper setup ip adress. About the port yes this is an error ;)

Comment (fe51) [reply pizero/runner.py]: Okay :)
Looks good to me then 👍

**Review by fe51 — APPROVED** [review]
Looks good to me now !
(Considering a proper set up for url etc will be implemented soon)

**Comment by fe51** [comment]
After fixing the linter :)

**Pull request merged**